container/go/fast_puller.go

```diff
@@ -0,0 +1,47 @@
+package main
```

Comment:
Add a top level doc for what this binary does. See [here](https://github.com/bazelbuild/rules_docker/blob/aafaf340e545322918aac22729ff11638b8c2452/container/go/cmd/extract_config/main.go#L1) for an example.